RampUpAuto/src/test/java/seleautomation/com/RampUpAuto/TitleSearch.java

```diff
@@ -0,0 +1,14 @@
+package RampUpAuto.src.test.java.seleautomation.com.RampUpAuto;
+ import org.testng.Assert;
+import org.openqa.selenium.WebDriver;
+
+public class TitleSearch {
+
+    public void AssertionTitleResult_(WebDriver Driver) {
+        String pageTitle = Driver.getTitle();
+        String expectedTitle = "Selenium Tutorials";
+        Assert.assertEquals("Selenium Tutorials", expectedTitle, pageTitle);
```

Comment:
Here the `Assert.assertEquals()` takes 3 parameters , the first one should be the actual string , the second one should be the expected string and the third one should be the message in case the assertion was wrong , but in your case you are comparing the "Selenium Tutorials" with expectedtitle so you should do sth like this 
  Assert.assertEquals(pageTitle, expectedTitle, "The title is wrong !");

Reply:
I will update it. 
Thanks